proxy/proxy.go

```diff
@@ -467,6 +467,33 @@ func setRequestURLForDynamicBackend(u *url.URL, stateBag map[string]interface{})
 	}
 }
 
+func normalizeSchemeHost(s, h string) (string, string, error) {
```

Comment:
There were 2 copies of this function before PR. After this PR in the current state there will be 4(!) copies of it. I better find some way to reuse it.

Reply:
Resolved in scope of https://github.com/zalando/skipper/pull/2824